governance/xc_admin/packages/xc_admin_frontend/components/tabs/Proposals.tsx

```diff
@@ -356,6 +379,275 @@ const Proposal = ({
                 </div>
               </>
             ) : null}
+            {instruction instanceof WormholeMultisigInstruction ? (
+              <div className="col-span-4 my-2 space-y-4 bg-darkGray2 p-4 lg:col-span-3">
+                <h4 className="h4">Wormhole Instructions</h4>
+                <hr className="border-[#E6DAFE] opacity-30" />
+                {instruction.governanceAction ? (
+                  <div className="base16 flex justify-between">
+                    <div>Governance Action</div>
+                    <div>
+                      {instruction.governanceAction instanceof ExecutePostedVaa
+                        ? 'Execute posted VAA'
+                        : 'Unknown governance action'}
+                    </div>
+                  </div>
+                ) : (
+                  <div className="base16 flex justify-between">
+                    <div>Governance Action</div>
+                    <div>Unknown wormhole message</div>
+                  </div>
+                )}
+                {instruction.governanceAction instanceof ExecutePostedVaa
+                  ? instruction.governanceAction.instructions.map(
+                      (instruction, index) => {
+                        const multisigParser = MultisigParser.fromCluster(
```

Comment:
Rename inner `multisigParser` to something else

Reply:
why?